hadoop-ozone/integration-test/src/test/java/org/apache/hadoop/ozone/om/snapshot/TestOzoneManagerSnapshotProvider.java

```diff
@@ -115,14 +120,26 @@ public void testDownloadCheckpoint() throws Exception {
             "does not match its ratis snapshot index");
   }
 
+  @SuppressFBWarnings("NP")
```

Comment:
Tried multiple things to avoid supressing but it didn't work. Apparently adding the new module is under LGPL license which fails as it's not defined under recognized license in license.sh. 
@adoroszlai @jojochuang any ideas to fix this.